controllers/testdata/declarative/case_5/expected/created/apiserver_deployment.yaml

```diff
@@ -96,6 +96,10 @@ spec:
               value: "ubi-minimal:test5"
             - name: MOVERESULTS_IMAGE
               value: "busybox:test5"
+            - name: METADATA_GRPC_SERVICE_SERVICE_HOST
+              value: ds-pipeline-metadata-grpc-testdsp5
+            - name: METADATA_GRPC_SERVICE_SERVICE_PORT
+              value: "1337"
```

Comment:
did you come up with this?? or is this default values from upstream? if former, can we pick a serious port, if it's out of scope then maybe a follow up task.

Reply:
it's the port used in the testdata's CR https://github.com/opendatahub-io/data-science-pipelines-operator/blob/main/controllers/testdata/declarative/case_5/deploy/cr.yaml#L17